packages/react-components/src/components/Attachment/AttachmentCard.tsx

```diff
@@ -181,6 +182,7 @@ const MappedMenuItems = (
     <TooltipHost content={menuActions[0].name}>
       <ToolbarButton
         aria-label={menuActions[0].name}
+        role="button"
```

Comment:
Does `ToolbarButton` not already have the requisite button functionality?
From https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role
Should the button functionality be wrapped, or is there something not enabled on the Fluent components?

Reply:
it doesn't seem to be the case
![image](https://github.com/Azure/communication-ui-library/assets/109105353/96441f1f-c936-42f7-8f5d-be237427300b)

Reply:
if look at the <button> it doesn't have a role specified

Reply:
But it is a `<button>`, which means we shouldn't need the role according to the linked doc:
> Adding role="button" tells assistive technology that the element is a button but provides no button functionality. Use [<button>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) or [<input>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) with type="button" instead.

Reply:
thanks you are right, l think a11y teams was complaining about how one of div is missing aria role not button, PR has been updated to remove that div

Reply:
<img width="1137" alt="image" src="https://github.com/Azure/communication-ui-library/assets/109105353/d142e54b-aa29-4a10-a270-6fd3a0d8d02b">